xbmc/cores/VideoPlayer/DVDCodecs/Overlay/DVDOverlaySSA.h

```diff
@@ -34,7 +34,6 @@ class CDVDOverlaySSA : public CDVDOverlay
   {
     replace = true;
     m_libass = libass;
-    libass->Acquire();
```

Comment:
I have not written this code so I can only guess. The subtitle parser owns the collection of line items. For rendering particular line items get cloned (for what reason ever). Looks kind of strange but I don't see an error there. In regard to libass cloning or increasing ref count directly is the same thing.

> never gets deleted.

this seems wrong. Items should be deleted with disposal of the collection.